### PR TITLE
minor fix to the css

### PR DESCRIPTION
**Pull request by devoncarew**
- fix a padding issue in the footer of pages

from:
screen shot 2017-09-20 at 1 13 09 pm

to:
screen shot 2017-09-20 at 1 14 44 pm

@jcollins-g

### CODE REVIEW

**Comment by kevmoo** [comment]
...although you need to update generated output to make Travis happy...

**Comment by kevmoo** [comment]
@devoncarew travis sad?

**Comment by devoncarew** [comment]
Yes, but a known issue: https://github.com/dart-lang/dartdoc/issues/1505

File: pubspec.yaml [hunk 1]
```diff
@@ -5,7 +5,7 @@ author: Dart Team <misc@dartlang.org>
 description: A documentation generator for Dart.
 homepage: https://github.com/dart-lang/dartdoc
 environment:
-  sdk: '>=1.23.0-dev.11.5 <2.0.0-dev.infinity'
+  sdk: '>=1.23.0-dev.11.5 <2.0.0'
```
Comment (jcollins-g) [inline pubspec.yaml]: ?

**Pull request merged**